test/views/resident/helpers/CheckYourAnswersPartialViewSpec.scala

```diff
@@ -81,10 +81,6 @@ class CheckYourAnswersPartialViewSpec extends CommonPlaySpec with WithCommonFake
         "has the question as part of the link" in {
           doc.select("#sellOrGiveAway-change-link a").text shouldBe s"${residentMessages.change} ${commonMessages.PropertiesSellOrGiveAway.heading}"
         }
-
-        "has the question component of the link as visuallyhidden" in {
-          doc.select("#sellOrGiveAway-change-link > span").text shouldBe commonMessages.PropertiesSellOrGiveAway.heading
```

Comment:
I don't think this should be removed. It can be fixed by updating the selector, there's still something with that ID, it's just one layer up in the html, so you'd need to first get the a tag and then the span.